secrets/api_secrets/tokens

```diff
@@ -0,0 +1,6 @@
+API_KEY=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
+API_KEY_TWO=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
+API_KEY_THREE=ghp_SerULubAEgmUvNss9EJMJLVYZ2Iprm29Mdlg
+API_KEY_FOUR=sl.BxYhe_DcMZdu0TiwIqjSxxWOG02QQ2aPk17N94bsnyPCNoZ1IbD5as-LUOFRF-t7qKJaoUUz_XpFxs1WFe9nZZXMkUxcXuyoV6Caje3E8Lyx55w0oTEo29Fpolfg8d_QEBIvqpJUiB9C
+API_KEY_FIVE=sl.BxPWLN25rQ2VVsFIy0AvjHY-Tiy1Pjuu9VZ-62fobHruF_5ANABMm48aG4Tjx6YohNF7mZia2zZVLbvqTlklsAQe41zzz0lqOp1dyCf-FFY-GFC9gHZ-gECgytQL8WW26j77W3jcS4ee
+API_KEY_SIX=glpat-QXnCVeWcFaUMRZM9sAUZ
```

Comment:
[comment]: <> (FrogbotReviewComment)


## 🤫 Secret Violation
<div align='center'>

| Severity                | ID                  | Finding                  | Watch Name                  | Policies                  |
| :---------------------: | :-----------------------------------: | :-----------------------------------: | :-----------------------------------: | :-----------------------------------: |
| ![high](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    High | EXP-1681-00004 | Hardcoded secrets were found | exp_watch8637 | exp_policy7427 |
| ![high](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    High | EXP-1681-00003 | Hardcoded secrets were found | shiftleft_watch | exp_p |
| ![high](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    High | EXP-1681-00002 | Hardcoded secrets were found | exp_watch8637 | exp_policy7427 |
| ![high](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    High | EXP-1681-00004 | Hardcoded secrets were found | shiftleft_watch_sec | exp_p |
| ![high](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    High | EXP-1681-00004 | Hardcoded secrets were found | shiftleft_watch | exp_p |
| ![high](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    High | EXP-1681-00001 | Hardcoded secrets were found | shiftleft_watch | exp_p |
| ![high](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    High | EXP-1681-00001 | Hardcoded secrets were found | exp_watch8637 | exp_policy7427 |
| ![high](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    High | EXP-1681-00003 | Hardcoded secrets were found | shiftleft_watch_sec | exp_p |
| ![high](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    High | EXP-1681-00003 | Hardcoded secrets were found | exp_watch8637 | exp_policy7427 |
| ![high](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    High | EXP-1681-00002 | Hardcoded secrets were found | shiftleft_watch_sec | exp_p |
| ![high](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    High | EXP-1681-00002 | Hardcoded secrets were found | shiftleft_watch | exp_p |
| ![high](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    High | EXP-1681-00001 | Hardcoded secrets were found | shiftleft_watch_sec | exp_p |

</div>


<details><summary><b>Full description</b></summary>

### Violation Details
|                 |                   |
| --------------------- | :-----------------------------------: |
| **Abbreviation:** | REQ.SECRET.GENERIC.TEXT |

Storing hardcoded secrets in your source code or binary artifact could lead to several risks.

If the secret is associated with a wide scope of privileges, attackers could extract it from the source code or binary artifact and use it maliciously to attack many targets. For example, if the hardcoded password gives high-privilege access to an AWS account, the attackers may be able to query/modify company-wide sensitive data without per-user authentication.

## Best practices

Use safe storage when storing high-privilege secrets such as passwords and tokens, for example -

* ### Environment Variables

Environment variables are set outside of the application code, and can be dynamically passed to the application only when needed, for example -
`SECRET_VAR=MySecret ./my_application`
This way, `MySecret` does not have to be hardcoded into `my_application`.

Note that if your entire binary artifact is published (ex. a Docker container published to Docker Hub), the value for the environment variable must not be stored in the artifact itself (ex. inside the `Dockerfile` or one of the container's files) but rather must be passed dynamically, for example in the `docker run` call as an argument.

* ### Secret management services

External vendors offer cloud-based secret management services, that provide proper access control to each secret. The given access to each secret can be dynamically modified or even revoked. Some examples include -

* [Hashicorp Vault](https://www.vaultproject.io)
* [AWS KMS](https://aws.amazon.com/kms) (Key Management Service)
* [Google Cloud KMS](https://cloud.google.com/security-key-management)

## Least-privilege principle

Storing a secret in a hardcoded manner can be made safer, by making sure the secret grants the least amount of privilege as needed by the application.
For example - if the application needs to read a specific table from a specific database, and the secret grants access to perform this operation **only** (meaning - no access to other tables, no write access at all) then the damage from any secret leaks is mitigated.
That being said, it is still not recommended to store secrets in a hardcoded manner, since this type of storage does not offer any way to revoke or moderate the usage of the secret.


<br></details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>